kitsune/search/v2/documents.py

```diff
@@ -140,6 +151,17 @@ class Index:
         name = config.QUESTION_INDEX_NAME
         using = config.DEFAULT_ES7_CONNECTION
 
+    @classmethod
+    def prepare(cls, instance):
+        """Override super method to exclude certain docs."""
+        # Add a discard field in the document if the following conditions are met
+        # Question document is spam
+        # TODO: figure out if archived questions are excluded too
+        if instance.is_spam:
+            instance.es_discard_doc = True
+
+        return super(QuestionDocument, cls).prepare(instance)
```

Comment:
`super()` should work, why go to all the trouble of upgrading to python 3 if we don't make use of it :stuck_out_tongue:

Reply:
I guess it's D.R.Y vs explicit here. In any case it's a matter of preference. After all super() does the same thing behind the scenes.

Fun-fact: supper = super and then call supper()

Reply:
:roll_eyes: